test/e2e/index.ts

```diff
@@ -603,28 +603,8 @@ test(
       installed_app_id,
       agent_key: agent,
       source: {
-        type: "bundle",
```

Comment:
Installing the happ from a manifest was redundant here anyway so I changed it to installing from path.